client/main.lua

```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
     DrawRect(0.0, 0.0+0.0125, 0.017+ factor, 0.03, 0, 0, 0, 75)
     ClearDrawOrigin()
 end
+
+function JailIntro()
+    local Ped = PlayerPedId()
```

Comment:
**NOT NECESSARY** but for a better code look this should all have the same name in script so `ped` and not `Ped`